### PR TITLE
7503_update_error_message

### PR DESCRIPTION
**Pull request by ozamani9gh**
*Issue #:* /bcgov/entity#7503

*Description of changes:*

Update Alterations error message.

### CODE REVIEW

**Review by severinbeauvais — COMMENTED** [review]
LGTM but you really need a BE developer's blessing.

**Review by JazzarKarim — APPROVED** [review]
Looks good to me!

**Comment by argush3** [comment]
Looks good, but can we fix the linting error?

File: legal-api/src/legal_api/services/filings/validations/alteration.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def company_name_validation(filing):
         validation_result = namex.validate_nr(nr_response)
 
         if not nr_response['requestTypeCd'] in ('CCR', 'CCP', 'BEC', 'BECV'):
-            msg.append({'error': babel('Alteration only available for Change of Name Name requests.'), 'path': nr_path})
+            msg.append({'error': babel('The name type associated with the name request number entered  \
+                                       cannot be used for this transaction type.'), 'path': nr_path})
```
Comment (argush3) [inline legal-api/src/legal_api/services/filings/validations/alteration.py:87]: I'm not sure this gives you the expected output.

Comment (ozamani9gh) [reply legal-api/src/legal_api/services/filings/validations/alteration.py:87]: Thanks Argus, yeah that makes sense, Thor also mentioned that i have blank spaces.... i have fixed it and will push my changes back up

**Review by argush3 — APPROVED** [review]

**Pull request merged**